.github/workflows/docker-build.yml

```diff
@@ -28,14 +28,12 @@ jobs:
         with:
           version: 'v0.9.1'
       - uses: actions/checkout@v4
-        with:
-          path: 'opensearch-benchmark-git'
+
       - name: Docker Build ${{ matrix.platform }}
         run: |
             docker buildx version
-            cp -a opensearch-benchmark-git/* ./
-            echo "Disable VERSION arg to enter docker build test mode"
-            PLATFORM=${{ matrix.platform }}
-            PLATFORM=`echo $PLATFORM | tr '/' '-'`
-            docker buildx build --platform ${{ matrix.platform }} --build-arg BUILD_ENV=testing --build-arg BUILD_DATE=`date -u +%Y-%m-%dT%H:%M:%SZ` -f "docker/Dockerfile" -t "osb/osb-$PLATFORM" -o type=docker .
-            docker images | grep "osb/osb-$PLATFORM"
+            tag=osb/osb-`echo ${{ matrix.platform }} | tr '/' '-'`
+            set -x
```

Comment:
I understand your changes here but this change combined with the Dockerfile change basically cause the docker-build check to be inaccurate.

It will only check docker build based on merged code, instead of the commit sent with the PR. You can only detect issues once you merge the code, which defeat the purpose of PR check.

My original design was meant to have test mode testing the PR commit, while prod mode use the original pypi method. I think we should bring back the test mode, but only change prod mode to build from source instead of pulling from pypi, which should be the best of both approaches. Thanks.

Reply:
The updated changes should pull in the PR commit now.